source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -341,6 +343,23 @@ You must add a line to your `composer.json` file if you have Composer-managed Wo
       "web/app/themes/{$name}/": ["type:wordpress-theme"]
     },
 	```
+## Known Issues and Workarounds
+### `terminus install:run <site>.<env> ocp` fails to install Object Cache Pro on _subdomain multisites_
+While most of the time, the Terminus command runs on single or WordPress multisites without issues, we have observed the workflow failing when _the Pantheon platform url_ (e.g. `dev-mysite.pantheonsite.io`) does not match the _site URL stored in the database_. This might happen if vanity domains are configured for the lower environments (e.g. `dev.mysite.com`). This can be observed when the link provided to the site from the Pantheon dashboard does not match the WordPress site URL. To check the URL stored in the database, you can run `terminus wp -- <site>.<env> option get siteurl`.
```

Comment:
```suggestion
While most of the time, the Terminus command runs on single or WordPress multisites without issues, we have observed the workflow failing when _the Pantheon platform url_ (e.g. `dev-mysite.pantheonsite.io`) does not match the _site URL stored in the database_. This might happen if vanity domains are configured for the non-live (e.g. `dev.mysite.com`). This can be observed when the link provided to the site from the Pantheon dashboard does not match the WordPress site URL. To check the URL stored in the database, you can run `terminus wp -- <site>.<env> option get siteurl`.
```

Not sure how common the use of "lower environments" is.

Reply:
We’ve been using “WordPress Multisites” with a capital M when describing the feature @jazzsequence

Reply:
didn't catch that one. will fix